src/main/kotlin/com/ajaxproject/telegrambot/bot/service/FinanceService.kt

```diff
@@ -1,44 +1,50 @@
 package com.ajaxproject.telegrambot.bot.service
 
-import com.ajaxproject.telegrambot.bot.models.MongoExpense
-import com.ajaxproject.telegrambot.bot.models.MongoIncome
+import com.ajaxproject.telegrambot.bot.dto.ExpenseResponse
+import com.ajaxproject.telegrambot.bot.dto.IncomeResponse
+import com.ajaxproject.telegrambot.bot.dto.toExpenseResponse
+import com.ajaxproject.telegrambot.bot.dto.toIncomeResponse
+import com.ajaxproject.telegrambot.bot.models.MongoFinance
 import com.ajaxproject.telegrambot.bot.repository.FinanceRepositoryImpl
-import org.bson.types.ObjectId
+import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
 
 @Service
 class FinanceService(
-    private val incomeRepositoryImpl: FinanceRepositoryImpl<MongoIncome>,
-    private val expenseRepositoryImpl: FinanceRepositoryImpl<MongoExpense>,
+    private val financeRepositoryImpl: FinanceRepositoryImpl,
 ) {
 
-    fun getIncomeByUserId(userId: Long): List<MongoIncome> {
-        return incomeRepositoryImpl.findByUserId(userId, MongoIncome::class.java, MongoIncome.COLLECTION_NAME)
+    fun getIncomeByUserId(userId: Long): List<IncomeResponse>? {
```

Comment:
```suggestion
    fun getAllIncomesByUserId(userId: Long): List<IncomeResponse> {
```
if methods return list, it should be named like getAll... instead of get...